### PR TITLE
Update security.md

### PR DESCRIPTION
**Pull request by RichiH**
Some minor cleanups and clarifications

### CODE REVIEW

File: content/docs/operating/security.md [hunk 1]
```diff
@@ -14,13 +14,15 @@ attack vectors that some configurations may enable.
...
-relevant component.
+relevant component. If you prefer to report security issues in secret, you may
+email the non-public [Prometheus team mailing list](mailto:prometheus-team@googlegroups.com).
 
 ### Prometheus
 
 It is presumed that untrusted users have access to the Prometheus HTTP endpoint
 and logs. They have access to all time series information contained in the
-database, plus a variety of operational/debugging information.
+database, plus a variety of operational/debugging information, including all
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: There's far more than that exposed - I don't think we should try to list it all.

File: content/docs/operating/security.md [hunk 2]
```diff
@@ -84,7 +86,8 @@ configuration file.
 Any user with access to the Pushgateway HTTP endpoint can create, modify and
 delete the metrics contained within. As the Pushgateway is usually scraped with
 `honor_labels` enabled, this means anyone with access to the Pushgateway can
-create any time series in Prometheus.
+create any time series in Prometheus, including impersonating other targets or
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: This isn't possible.

File: content/docs/operating/security.md [hunk 3]
```diff
@@ -14,13 +14,15 @@ attack vectors that some configurations may enable.
 
 As with any complex systems it is not possible to guarantee that there are no
 bugs. If you find a security bug, please file it in the issue tracker of the
-relevant component.
+relevant component. If you prefer to report security issues in secret, you may
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: I'd prefer not spam the team list with misunderstandings of our security model.

File: content/docs/operating/security.md [hunk 4]
```diff
@@ -34,10 +36,10 @@ that service discovery system.
 
 Scraped targets may be run by untrusted users. It should not by default be
 possible for a target to expose data that impersonates a different target.  The
-`honor_labels` option removes this protection, as can certain relabelling
-setups.
+`honor_labels` option removes and certain relabelling setups remove this
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: This change doesn't parse.

File: content/docs/operating/security.md [hunk 5]
```diff
@@ -34,10 +36,10 @@ that service discovery system.
...
+protection.
 
-As of Prometheus 2.0, the `--web.enable-admin-api` flag controls access to the
+As of Prometheus 2.x, the `--web.enable-admin-api` flag controls access to the
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: This is correct as written

File: content/docs/operating/security.md [hunk 6]
```diff
@@ -136,7 +139,7 @@ wherever it is stored.
 ## Denial of Service
 
 There are some mitigations in place for excess load or expensive queries.
-However, if too many or too expensive queries/metrics are provided components
+However, if too many or too expensive queries/metrics are provided, components
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: This is fine as-is.

File: content/docs/operating/security.md (hunk 3 above)
Comment (RichiH) [reply content/docs/operating/security.md]: While I agree with that sentiment, we need to have a defined way for non-public disclosure.

File: content/docs/operating/security.md (hunk 1 above)
Comment (RichiH) [reply content/docs/operating/security.md]: time series is already an incomplete list by itself. From experience, people care most about the three I now listed. I am 50/50 on this change, though.

File: content/docs/operating/security.md (hunk 4 above)
Comment (RichiH) [reply content/docs/operating/security.md]: Your phrasing is overly convoluted, I tried to untangle it.

Comment (RichiH) [reply content/docs/operating/security.md]: I am fine with different wording, of course.

File: content/docs/operating/security.md (hunk 5 above)
Comment (RichiH) [reply content/docs/operating/security.md]: I agree that it's correct, but "as of" is often misunderstood as "at this time" as opposed to "from this time on". The change would make it easier on non-native speakers. We could also phrase it as "Starting from 2.0 [...]".

File: content/docs/operating/security.md (hunk 2 above)
Comment (RichiH) [reply content/docs/operating/security.md]: How so? This was written in this file before my PR; see above.

File: content/docs/operating/security.md (hunk 6 above)
Comment (RichiH) [reply content/docs/operating/security.md]: I disagree.

File: content/docs/operating/security.md (hunk 3 above)
Comment (brian-brazil) [reply content/docs/operating/security.md]: We don't need one, we merely need to document how we accept security issues.

File: content/docs/operating/security.md (hunk 1 above)
Comment (brian-brazil) [reply content/docs/operating/security.md]: memory profiles, metadata, logs (at some point) - we'll keep on adding to this

File: content/docs/operating/security.md (hunk 4 above)
Comment (brian-brazil) [reply content/docs/operating/security.md]: Your wording still doesn't parse.

File: content/docs/operating/security.md (hunk 5 above)
Comment (brian-brazil) [reply content/docs/operating/security.md]: It's as of 2.0, not 2.x.

File: content/docs/operating/security.md (hunk 2 above)
Comment (brian-brazil) [reply content/docs/operating/security.md]: You cannot generate alerts this way. The current wording is fine.

File: content/docs/operating/security.md (hunk 6 above)
Comment (brian-brazil) [reply content/docs/operating/security.md]: "However components will fall over" does not make sense.

**Comment by beorn7** [comment]
The file got reworked a bit independently. I close this as obsolete.

**Pull request closed without merging**